apps/editing-toolkit/editing-toolkit-plugin/readme.txt

```diff
@@ -1,9 +1,9 @@
 === WordPress.com Editing Toolkit ===
 Contributors: alexislloyd, allancole, automattic, bartkalisz, codebykat, copons, dmsnell, get_dave, glendaviesnz, gwwar, iamtakashi, iandstewart, jeryj, Joen, jonsurrell, kwight, marekhrabe, mattwiebe, mkaz, mmtr86, mppfeiffer, noahtallen, nosolosw, nrqsnchz, obenland, okenobi, owolski, philipmjackson, vindl
 Tags: block, blocks, editor, gutenberg, page
-Requires at least: 5.0
-Tested up to: 5.5
-Stable tag: 2.9
+Requires at least: 5.5
+Tested up to: 5.6
```

Comment:
I updated our required WordPress versions too. This just matches what we were already doing anyways.